src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_index_pattern/__tests__/step_index_pattern.test.js

```diff
@@ -23,166 +20,75 @@ const savedObjectsClient = {
 };
 const goToNextStep = () => {};
 
-describe('StepIndexPattern', () => {
-  it('should render normally', () => {
-    const component = shallow(
-      <StepIndexPattern
-        allIndices={allIndices}
-        isIncludingSystemIndices={false}
-        esService={esService}
-        savedObjectsClient={savedObjectsClient}
-        goToNextStep={goToNextStep}
-      />
-    );
-
-    expect(component).toMatchSnapshot();
-  });
-
-  it('should render the loading state', () => {
-    const component = shallow(
-      <StepIndexPattern
-        allIndices={allIndices}
-        isIncludingSystemIndices={false}
-        esService={esService}
-        savedObjectsClient={savedObjectsClient}
-        goToNextStep={goToNextStep}
-      />
-    );
-
-    component.setState({ query: 'k', isLoadingIndices: true });
-
-    expect(component).toMatchSnapshot();
-  });
-
-  it('should render some indices', async () => {
-    const component = shallow(
-      <StepIndexPattern
-        allIndices={allIndices}
-        isIncludingSystemIndices={false}
-        esService={esService}
-        savedObjectsClient={savedObjectsClient}
-        goToNextStep={goToNextStep}
-      />
-    );
-
-    const instance = component.instance();
+const createComponent = props => {
+  return shallow(
+    <StepIndexPattern
+      allIndices={allIndices}
+      isIncludingSystemIndices={false}
+      esService={esService}
+      savedObjectsClient={savedObjectsClient}
+      goToNextStep={goToNextStep}
+      {...props}
+    />
+  );
+};
 
-    await instance.onQueryChanged({
-      target: { value: 'k' }
+describe('StepIndexPattern', () => {
+  describe('step 1', () => {
+    it('renders the loading state', () => {
+      const component = createComponent();
+      component.setState({ isLoadingIndices: true });
+      expect(component.find('[data-test-subj="createIndexPatternStep1Loading"]')).toMatchSnapshot();
```

Comment:
Hmm. So it looks like you changed these to not look at the entire component, but rather a subset of the component. I'm wondering if that's what we want since we don't have a single test ensuring the overall structure is maintained (like the component wrapped in a `<EuiPanel>` or the ordering makes sense). Can you elaborate more on what you're thinking with this change?